runtime/makelib/targets.mk.ftl

```diff
@@ -531,7 +528,7 @@ cnathelp$(UMA_DOT_O):cnathelp.cpp
 	$(CXX) $(CXXFLAGS) -mstackrealign -c $<
 </#if>
 
-# JIT helper methods require us to disable buffer security checks on Windows
+# special handling for cnathelp.cpp to disable buffer security check in windows for JIT helpers
```

Comment:
Please revert this change as it's not ztpf related.

Reply:
Reverted.

Reply:
it doesn't look reverted